challenges/backend/src/app/services/CarOnSaleClient/classes/CarOnSale.ts

```diff
@@ -0,0 +1,113 @@
+import { AxiosInstance } from "axios";
+import { inject, injectable } from "inversify";
+import { DependencyIdentifier } from "../../../DependencyIdentifiers";
+import { IConfig } from "../../Config/interface/IConfig";
+import { ILogger } from "../../Logger/interface/ILogger";
+import { Auction } from "../dtos/auction/auction";
+import { AuctionItem } from "../dtos/auction/AuctionItem";
+import { AuthenticationResponse } from "../dtos/auth/AuthenticationResponse";
+import { IActionFilter } from "../interface/IActionFilter";
+import { ICalculator } from "../interface/ICalculator";
+import { ICarOnSaleClient } from "../interface/ICarOnSaleClient";
+
+@injectable()
+export class CarOnSale implements ICarOnSaleClient {
+    constructor(@inject(DependencyIdentifier.LOGGER) private logger: ILogger,
+        @inject(DependencyIdentifier.CLIENT) private readonly axiosInstance: AxiosInstance,
+        @inject(DependencyIdentifier.CONFIG) private readonly config: IConfig,
+        @inject(DependencyIdentifier.CALCULATOR) private readonly calculator: ICalculator
+    ) {
+
+    }
+    async print(): Promise<void> {
```

Comment:
The class method 'print' must be marked either 'private', 'public', or 'protected'